components/core/Test.py

```diff
@@ -0,0 +1,16 @@
+from UserManager import * 
```

Comment:
We already have a components/core/test directory, let's contain all the test case there.

Have meaningful test file names. eg: test_login.py

Reply:
done